subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c

```diff
@@ -460,7 +460,7 @@ uint32_t lll_preempt_calc(struct ull_hdr *ull, uint8_t ticker_id,
 	uint32_t diff;
 
 	ticks_now = ticker_ticks_now_get();
-	diff = ticks_now - ticks_at_event;
+	diff = ticker_ticks_diff_get(ticks_now, ticks_at_event);
```

Comment:
This change is duplicated in PR #52124, undo this change here

Reply:
Without this commit, the bsim_bt_multiple test asserts. When #52124 is merged I will rebase this PR.